src/content/docs/iast/install.mdx

```diff
@@ -9,44 +9,179 @@ freshnessValidatedDate: never
 ---
 
 <Callout variant="important">
-  Run IAST with non-production deployments only. IAST tests by invoking HTTP requests with an exploit payload. You must use IAST in non-production environments with only simulated data to avoid both data corruption and introducing exploits into your live code. In addition, when you run IAST on new code in pre-production, you catch potential vulnerabilities before they go live.
+  Run IAST with non-production deployments only. IAST tests the application by
+  invoking HTTP requests with an exploit payload. You must use IAST in
+  non-production environments with only simulated data to avoid both data
+  corruption and introducing exploits into your live code. In addition, when you
+  run IAST on new code in pre-production, you catch potential vulnerabilities
+  before they go live.
 </Callout>
 
+## Install New Relic IAST
+
+There are primarily five stages involved in successful installation of IAST:
```

Comment:
Many of the screenshots in this article dont seem to add value. Discuss with me.